lib/generator/template_values.rb

```diff
@@ -1,24 +1,30 @@
 module Generator
   # Contains methods accessible to the ERB template
   class TemplateValues
-    attr_reader :abbreviated_commit_hash, :version, :test_cases
+    attr_reader :abbreviated_commit_hash, :version, :exercise_name, :test_cases
```

Comment:
Now there is only one of them per *_cases.rb

I'm probably just exchanging one type of complexity  for another.